packages/central-server/src/apiV2/feedItems/EditFeedItems.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import { BESAdminEditHandler } from '../EditHandler';
+import { uploadImage } from '../utilities';
+/**
+ * Handles PUT endpoints:
+ *  - /feedItems
+ */
+
+export class EditFeedItems extends BESAdminEditHandler {
+  async getFields() {
+    const feedItem = await this.models.feedItem.findById(this.recordId, {
+      columns: ['template_variables'],
+    });
+    return feedItem;
+  }
+
+  async updateRecord() {
+    const {
+      template_variables: {
+        title: updatedTitle,
+        image: base64Image,
+        body: updatedBody,
+        link: updatedLink,
+      },
+    } = this.updatedFields;
+    const updatedFields = { ...this.updatedFields };
+
+    const {
+      template_variables: {
+        title: existingTitle,
+        image: existingImage,
+        body: existingBody,
+        link: existingLink,
+      },
+    } = await this.getFields();
+
+    if (base64Image !== undefined) {
+      updatedFields.template_variables = {
+        title: updatedTitle || existingTitle,
+        image: await uploadImage(base64Image, existingImage),
```

Comment:
The params here don't quite match the function.

```
export const uploadImage = async (
  encodedImage,
  uniqueId,
  imageSuffix,
  useTimestamp = false,
  existingImagePath,
)
```

From glancing at that function I think all your images are going to be uploaded with the name `undefined_undefined` which is probably not what we want. Let's add `uniqueId` and `imageSuffix` values to both uses of `uploadImage`